### PR TITLE
[CDAP-18916] [CDAP-18917] Added ANY selector for the Deduplicate operation 

### PR DESCRIPTION
**Pull request by fernst**
Added deprecate notice for the FIRST and LAST operations for the Deduplicate aggregator in the UI and in the docs.

### CODE REVIEW

File: core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/function/AnyTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package io.cdap.plugin.batch.aggregator.function;
+
+import io.cdap.cdap.api.data.schema.Schema;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+/**
+ *
+ */
```
Comment (sanjanasandeep) [inline core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/function/AnyTest.java:26]: needs description?

File: core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/function/AnyTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2016-2019 Cask Data, Inc.
+ *
```
Comment (sanjanasandeep) [inline core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/function/AnyTest.java:3]: should this be 2022? or the license must be copied as is?

File: core-plugins/docs/Deduplicate-batchaggregator.md [hunk 3]
```diff
@@ -4,8 +4,12 @@
 Description
 -----------
 De-duplicates records either using one or more fields or by using the record as a whole. Additionally, it supports logically
-choosing a record out of the duplicate records based on a filter field. Supported logical functions are `max`, `min`,
-`first`, and `last`.
+choosing a record out of the duplicate records based on a filter field. Supported logical functions are `any`, `max` and `min`. 
+
+The `first` and `last` are deprecated and will be removed in later versions of the platform.
+
```
Comment (sanjanasandeep) [inline core-plugins/docs/Deduplicate-batchaggregator.md:10]: should we add a reason why this is deprecated?

Comment (fernst) [reply core-plugins/docs/Deduplicate-batchaggregator.md:10]: No

File: core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/function/AnyTest.java (hunk 2 above)
Comment (fernst) [reply core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/function/AnyTest.java:3]: oops, let me fix

**Review by tivv — APPROVED** [review]

**Review by sanjanasandeep — APPROVED** [review]

File: core-plugins/docs/Deduplicate-batchaggregator.md [hunk 4]
```diff
@@ -4,8 +4,12 @@
 Description
 -----------
 De-duplicates records either using one or more fields or by using the record as a whole. Additionally, it supports logically
-choosing a record out of the duplicate records based on a filter field. Supported logical functions are `max`, `min`,
-`first`, and `last`.
+choosing a record out of the duplicate records based on a filter field. Supported logical functions are `any`, `max` and `min`. 
+
+The `first` and `last` are deprecated and will be removed in later versions of the platform.
```
Comment (rrielley76) [inline core-plugins/docs/Deduplicate-batchaggregator.md:9]: @sanjanasandeep Can we add logical functions after first and last?  "The 'first' and 'last' logical functions are deprecated...."

Comment (rrielley76) [inline core-plugins/docs/Deduplicate-batchaggregator.md:9]: @sanjanasandeep Can this be a Note at the end of the Description?

**Pull request merged**